portfolio.html

```diff
@@ -0,0 +1,96 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+	<title>Max G Portfolio</title>
+	<link rel="Stylesheet" type="text/css" href="portfolio.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
would be worth running the html through HTML validator if you haven't already. In this case I think you should add the following element
`<meta charset="utf-8">`